config/settings.yml

```diff
@@ -89,7 +110,8 @@ test_sqlite:
 # MUST BE SET FOR DEVELOPMENT & PRODUCTION
 # Example generated with SecureRandom.base64(32)
 oauth2:
-  key: "IeGpe9iq27Glqujd8pGH1+xM8C/ofm97Lp74N+6cN0I="
+  #local key: "IeGpe9iq27Glqujd8pGH1+xM8C/ofm97Lp74N+6cN0I="
+  key: 'znVwVn9oSKPIZhzr+7oSR2rk4dONsQyjH0EeW++uCY8'
```

Comment:
@frankiepun - Did you intend to change this default key?

Reply:
Sorry, my fault. I removed the bad key for testing